packages/tentales/src/middleware/middlewares/__tests__/error-middleware.test.ts

```diff
@@ -0,0 +1,215 @@
+/* tslint:disable no-any no-empty */
+
+import Koa from "koa"
+import supertest from "supertest"
+import { errorMiddleware } from "../error-middleware"
+import { createTestMw, createTestMwConfig } from "../../../__test-helpers__"
+import { AuthError } from "../../errors"
+
+describe("Incoming request", () => {
+  test("It does not do anything on the way down", async () => {
+    // Setup
+    const app = new Koa()
+    app.use(errorMiddleware(createTestMwConfig()))
+    app.use(createTestMw())
+    const request = supertest.agent(app.listen())
+
+    // Perform
+    return request.get(`/foo`).expect(200, "Next body")
```

Comment:
What is the second arg to `expect` here, "Next body"?

Reply:
Body content of the response. It's supertest's shorthand function.

Reply:
`Next body` is the body set by the next middleware :)

Reply:
Aha, don't see the next middleware, that throw me off.